<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionSettingsDialog</class>
 <widget class="QDialog" name="AcquisitionSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Acquisition Settings</string>
  </property>
  <widget class="QCheckBox" name="time_points_check_box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables time points (time series). If unchecked, acquistion script will only be performed once. If checked, the acquisition script will be repeated Count number of times with an interval of Interval minutes between them.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Time points</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num_time_points_line_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of time points in time series&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_points_interval_line_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Interval between time points in minutes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="channel_order_move_up_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Moves channel up in channel order list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="channel_order_move_down_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Moves channel down in channel order list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
  </widget>
  <widget class="QListView" name="channel_order_list_view">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Order of channels in Z-stack, Snap, and Video acquisitions from top to bottom&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="channel_order_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel Order</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="count_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Count:</string>
   </property>
  </widget>
  <widget class="QLabel" name="interval_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interval:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse and choose directory for acquisition to be saved&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="save_path_line_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save location of acquisition&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="save_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Location:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_acquisition_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Starts acquisition from settings set here and the region setup dialog. Also brings up the Acquisition Dialog which will provide updates during the acquisition and has an option to abort the acquisition.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Start Acquisition</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_images_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Images:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_images_line_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total number of images taken during the entire acquisition&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="memory_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Memory:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="memory_line_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total amount of data taken during the entire acquisition&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_images_per_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Images per point:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num_images_per_line_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of images  per time point&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>20</width>
     <height>491</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="acq_settings_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Acquisition Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>50</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="memory_unit_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gb</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="researcher_line_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name of researcher&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="researcher_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Researcher:</string>
   </property>
  </widget>
  <widget class="QLabel" name="save_acquisition_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save/Acquisition Start</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="interval_label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_acquisition_dialog_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Starts acquisition from settings set here and the region setup dialog. Also brings up the Acquisition Dialog which will provide updates during the acquisition and has an option to abort the acquisition.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Show Acquisition Dialog</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>270</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="adv_settings_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Moves channel down in channel order list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Advanced Acquisition Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="adv_settings_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
